app/source/css/components/PageMenu.styl

```diff
@@ -37,19 +37,18 @@
           font-size: 15px;
         }
 
-          &:hover {
-             border-bottom: 4px solid white;
-             padding-bottom: 14px !important;
-           }
-             
-           &.active {
-             a {
-               border-bottom: 4px solid;
-               padding-bottom: 14px !important;
-               outline: none;
-             }  
-           } 
-      } 
+        hover() {
+          border-bottom: 4px solid white;
+          padding-bottom: 14px !important;
```

Comment:
Got it. Ok, I'll maybe just spot check the places I'm aware of by a global search, and then at mention y'all on said PR—since I obviously don't know about the edges of the Loconomics "app universe" so to speak.